troposphere/static/js/components/modals/link/ExternalLinkDeleteModal.react.js

```diff
@@ -8,7 +8,7 @@ export default React.createClass({
     mixins: [BootstrapModalMixin],
 
     propTypes: {
-      link: React.PropTypes.instanceOf(Backbone.Model).isRequired
+          external_link: React.PropTypes.instanceOf(Backbone.Model).isRequired
```

Comment:
I was going to fix it in this pr, but there are *many* places where it needs to be updated.